massgov/openmass
# Dp 25112 add external organization field to external link content type

**Description:**
      - Added field_external_organization to external_data_resrouce
      - If field_external_organization has a value, replaces the organization shown on the collection listing pages.

**Jira:** (Skip unless you are MA staff)
DP-25112

**To Test:**
- Create/Choose one new term in collections.
- Create/Choose an organization.
- Create an External link for Collections with the collection term and the organization you chose.
- Visit the collection term.
- Notice the organization from the Organizations field is shown on the page.
- Edit the External Link for Collections again, and enter a value for "External organization"
- Visit the collection term page again and notice the external organization shows up.
- Change again the "External organization" to only spaces. Save.
- Visit the term page again and notice the "External organization" (only spaces) is NOT showing up on the collection term page.

---

[Peer Review Checklist](https://github.com/massgov/openmass/blob/develop/docs/peer_review_checklist.md)
